### PR TITLE
Merge Release branch to main

### PR DESCRIPTION
**Pull request by asifdxtreme**
Merge release branch to main 
This merge brings in the changes for 0.7.0 release and add support for validating webhook for CSI drivers.

### CODE REVIEW

File: artifacts/compatibility-yaml/compatibility-v1.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/namespace.yaml",
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/webhook.yaml",
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/vsphere-csi-driver.yaml" ]
+    },
+    "2.6.2": {
+      "vSphere": { "min": "6.7.1", "max": "8.0.1" },
+      "k8s": { "min": "1.22", "max": "1.24" },
+      "isCPIRequired": false,
+      "deploymentPath": [
+          "https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/v2.6.2/manifests/vanilla/namespace.yaml",
```
Comment (ridaz) [inline artifacts/compatibility-yaml/compatibility-v1.0.0.yaml:26]: Why don't we need the webhook.yaml or validatingwebhook.yaml for CSI 2.6.2, similar to other versions?

Comment (asifdxtreme) [reply artifacts/compatibility-yaml/compatibility-v1.0.0.yaml:26]: Because this was introduced from 2.7 onwards

**Review by ridaz — APPROVED** [review]
LGTM, Thanks

**Review by jvrahav — CHANGES_REQUESTED** [review]
@asifdxtreme can you please add description to this change. what change does this bring in, if there are any issues that this solves

**Comment by asifdxtreme** [comment]
> @asifdxtreme can you please add description to this change. what change does this bring in, if there are any issues that this solves

Updated the description.

File: artifacts/compatibility-yaml/compatibility-v0.7.0.yaml [hunk 2]
```diff
@@ -1,14 +1,22 @@
 {
   "CSI" : {
+    "3.0.0" : {
+      "vSphere": { "min": "6.7.1", "max": "8.2.0" },
```
Comment (jvrahav) [inline artifacts/compatibility-yaml/compatibility-v0.7.0.yaml:4]: is this validated to work in vsphere 6.7.1

looks like 3.0.0 release carries certain features that only work from 8.0U3 onwards

**Comment by jvrahav** [comment]
> > @asifdxtreme can you please add description to this change. what change does this bring in, if there are any issues that this solves
> 
> Updated the description.
Thanks, @asifdxtreme can you also update the testing done for this PR

**Comment by asifdxtreme** [comment]
We don't have infra to test, this branch is already released in IBM in the starting of this year and customers are using it from last 7-8 months with no issues reported so far.

File: artifacts/compatibility-yaml/compatibility-v0.7.0.yaml (hunk 2 above)
Comment (asifdxtreme) [reply artifacts/compatibility-yaml/compatibility-v0.7.0.yaml:4]: The basic features still works in the older version as well. the combined compatibility with K8s and vsphere will any how not allow to run this on older than 7.x

**Review by jvrahav — APPROVED** [review]

**Pull request merged**